README.md

```diff
@@ -1,8 +1,8 @@
 ![](https://img.shields.io/badge/Microverse-blueviolet)
 
-# Project Name
+# World Bank GDP App
 
```

Comment:
- [ ] [OPTIONAL] i think it is better to add a screenshot of your project in your README file